### DISCUSSION TITLE
Equipo de trabajo Hackaton

### DISCUSSION THREAD

**Discussion by Tintto** [Launch-X-Latam/MisionFrontEnd#558] [Team Posts]
Hola chicos, algún equipo con espacio de participantes o alguien que se quiera unir??

**Comment by Javilh97** [comment]
Hola, si gustas podemos hacer equipo :D

**Comment by Tintto** [comment]
Hola , mi correo es lx11061@innovaccion.mx... Bienvenidos los que se quieran unir !!

**Comment by davidjmcxl** [comment]
Buenas noches busco team para hackaton .
Agg :+573017582116

**Comment by JulianVidal01** [comment]
Hola, Te escribí por teams por si todavía existe cupo

**Comment by innovategm** [comment]
aun les falta integrante? 
lx05800@innovaccion.mx

**Comment by davidjmcxl** [comment]
Buenos días amigos busco un equipo para la hackaton ,me pueden contactar
por ***@***.***
